src/Grid/topology.jl

```diff
@@ -86,12 +110,31 @@ function _get_facet_facet_neighborhood(::ExclusiveTopology, #=rdim=#::Val{:mixed
     Access the `vertex_vertex_neighbor`, `edge_edge_neighbor`, or `face_face_neighbor` fields explicitly instead."))
 end
 
-function Base.show(io::IO, ::MIME"text/plain", topology::ExclusiveTopology)
-    println(io, "ExclusiveTopology\n")
-    print(io, "  Vertex neighbors: $(size(topology.vertex_vertex_neighbor))\n")
-    print(io, "  Face neighbors: $(size(topology.face_face_neighbor))\n")
-    println(io, "  Edge neighbors: $(size(topology.edge_edge_neighbor))")
+# Guess of how many neighbors depending on grid dimension and index type.
+# Better to guess a bit too high than a bit too low.
```

Comment:
```suggestion
# Guess of how many neighbors depending on grid dimension and index type. 
# This could be possible to optimize further by studying connectivities of non-uniform
# grids, see https://github.com/Ferrite-FEM/Ferrite.jl/pull/974#discussion_r1660838649
```